### ISSUE TITLE
CORS Setting - Debug / Issues

### ISSUE THREAD

**Issue by kayvanaarssen** [pglombardo/PasswordPusher#1545]
Like requested in: https://github.com/pglombardo/PasswordPusher/issues/1193
Hereby an new issue with the details summarized.
I'll only copy and past the post from the above source. in here for quick reference.

@pglombardo I'll try to setup a clone of our existing one and test like I did before and let you know.

@pglombardo With the dadup service its still not working... But that could be because of the CORS settings on their side. Going to test backblaze now.

For some reason with backblaze even after setting the CORS rules i still cannot push. 
Get this error in console;

```
Access to XMLHttpRequest at 'https://pwpush.s3.us-west-000.backblazeb2.com/nu6pwcsx?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=074547e490ae%2F20231011%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231011T155449Z&X-Amz-Expires=300&X-Amz-SignedHeaders=content-length%3Bcontent-md5%3Bcontent-type%3Bhost&X-Amz-Signature=1967e94a93f0ac931b212c733d66e368b08471cb8cf6fd9c7b33394b189b1996' from origin 'https://test.domain.nl' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
 ```

![image](https://github.com/pglombardo/PasswordPusher/assets/672841/c99d9b43-c9e3-475f-8535-401dff3ebfb4)
